docs/package.json

```diff
@@ -13,8 +13,8 @@
     "export": "rimraf docs/export && next export --threads=3 -o export && yarn build-sw",
     "icons": "rimraf public/static/icons/* && node ./scripts/buildIcons.js",
     "typescript": "tsc -p tsconfig.json",
-    "typescript:transpile": "node scripts/formattedTSDemos",
-    "typescript:transpile:dev": "node scripts/formattedTSDemos --watch"
+    "typescript:transpile": "cross-env BABEL_ENV=development babel-node --extensions \".tsx,.ts,.js\" scripts/formattedTSDemos",
```

Comment:
Why do you need to switch to `babel-node` here? I would recommend to stick to just node until it becomes absolutely necessary.

Reply:
If I leave it as it was the script fails. I checked how it's done in the core and copied the approach.